letsdrill.rb

```diff
@@ -1,12 +1,25 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  grade = ''
+  case integer
```

Comment:
the case statement returns whatever the result of whatever meets the condition so no need to use the `grade` variable anywhere within this function.

see: https://ruby-doc.org/docs/keywords/1.9/Object.html#method-i-case